src/PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp

```diff
@@ -53,6 +52,9 @@ struct SpacetimeNormalVector;
 template <size_t Dim, typename Frame = Frame::Inertial,
           typename DataType = DataVector>
 struct TraceSpacetimeChristoffelFirstKind;
+template <size_t Dim, typename Frame = Frame::Inertial,
+          typename DataType = DataVector>
+struct TraceSpatialChristoffelFirstKind;
```

Comment:
Why don't SpatialChristoffelFirstKind and SpatialChristoffel2ndKind appear in TagDeclarations.hpp?

Reply:
They already exist in [TagsDeclaration.hpp](https://github.com/sxs-collaboration/spectre/blob/f23b41e5afc3947d7d2d6a6f74f2bb8d6aa990b8/src/PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp#L42) which is why they weren't added in this PR.